### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: tests/smoke/test_profile.py [hunk 1]
```diff
@@ -83,3 +83,32 @@ def test_exops_alerts_anon_user_302():
...
+
+
+def test_directory_supplier_verified_user():
+    token = 'Bearer {token}'.format(token=users['verified']['token'])
+    headers = {'Authorization': token}
+    url = get_absolute_url('profile:directory-supplier')
+    response = requests.get(url, headers=headers)
+
+    assert response.status_code == http.client.OK
+    assert response.json()['sso_id'] == users['verified']['sso_id']
```
Comment (konradko) [inline tests/smoke/test_profile.py]: should also check that the company profile returned contains the expected data

File: tests/smoke/test_profile.py [hunk 2]
```diff
@@ -83,3 +83,32 @@ def test_exops_alerts_anon_user_302():
...
+
+
+def test_directory_supplier_unverified_user():
+    token = 'Bearer {token}'.format(token=users['unverified']['token'])
+    headers = {'Authorization': token}
+    url = get_absolute_url('profile:directory-supplier')
+    response = requests.get(url, headers=headers)
+
+    assert response.status_code == http.client.OK
+    assert response.content == b''
```
Comment (konradko) [inline tests/smoke/test_profile.py]: feels like a 404 would be better, or at least a dict with `sso_id` and empty company profile field

Comment (sdonk) [inline tests/smoke/test_profile.py]: This needs merging then https://github.com/uktrade/directory-sso-profile/pull/48/files

@konradko

File: tests/smoke/test_profile.py [hunk 3]
```diff
@@ -83,3 +83,39 @@ def test_exops_alerts_anon_user_302():
...
+
+
+def test_directory_supplier_verified_user():
+    token = 'Bearer {token}'.format(token=users['verified']['token'])
+    headers = {'Authorization': token}
+    url = get_absolute_url('profile:directory-supplier')
+    response = requests.get(url, headers=headers)
+
+    assert response.status_code == http.client.OK
+    assert response.json() == {
```
Comment (konradko) [inline tests/smoke/test_profile.py]: that's odd, looks like this company doesn't really have a profile? Company number, name and industries are required AFAIK.

Comment (sdonk) [inline tests/smoke/test_profile.py]: Not in the model, so I went for the path of least resistance and didn't add them :D

Comment (konradko) [inline tests/smoke/test_profile.py]: Let's test for the correct situation, so company with number etc :-)

**Pull request merged**